Zend/zend_execute.c

```diff
@@ -2598,36 +2595,30 @@ static zend_always_inline zend_bool check_type_stdClass_assignable(zend_type typ
 
 /* Checks whether an array can be assigned to the reference. Returns conflicting property if
  * assignment is not possible, NULL otherwise. */
-static zend_always_inline zend_property_info *i_zend_check_ref_array_assignable(zend_reference *ref) {
+static zend_never_inline zend_bool zend_check_ref_array_assignable(zend_reference *ref) {
```

Comment:
The convention for these function is that "check" only checks something and "verify" also throws an error. So these should be renamed to `zend_verify_ref_*` now.